tls/cfsslscan_handshake.go

```diff
@@ -2,9 +2,58 @@ package tls
 
 import "errors"
 
+// IsECcipherSuite returns whether a given uint16 is code for an
+// EC-based TLS cipher suite. Comments indicate non-EC ranges.
+// encodings are defined by IANA:
+// http://www.iana.org/assignments/tls-parameters/tls-parameters.xml
+func IsECcipherSuite(cs uint16) bool {
```

Comment:
I think I'd prefer there in the `CipherSuites` lookup table rather than as a function. i.e. add an `CipherSuite.EllipticCurve` field like the `CipherSuite.ForwardSecret` field, or the `suiteECDHE` flag in the [crypto/tls](https://github.com/cloudflare/cf-tls/blob/master/tls/cipher_suites.go#L38) cipher suites.

In fact, `crypto/tls` is probably best with just a generic flags field like that has possible values from constants `EllipticCurve`, `ForwardSecret`, etc.